examples/OpenSSL-asm/config.cmake

```diff
@@ -0,0 +1 @@
+hunter_config(OpenSSL VERSION 1.0.2p CMAKE_ARGS ASM_SUPPORT=ON CMAKE_C_FLAGS=-fPIC configure_opts=no-dso)
```

Comment:
User config should be minimal, I don't think anything except `ASM_SUPPORT=ON` is required.

```cmake
hunter_config(
    OpenSSL
    VERSION ${HUNTER_OpenSSL_VERSION}
    CMAKE_ARGS ASM_SUPPORT=ON
)
```
Should "just work".